docs/content/tutorial_code/getting_started_1.md

```diff
@@ -1,7 +1,321 @@
 ---
-title: "Getting Started with WebAssembly"
+title: "Getting Started with WebAssembly Hub"
 weight: 1
 description: Create a simple WebAssembly filter and run it with Envoy
 ---
 
-TODO
+In this tutorial we'll take a look at creating a new WebAssembly (WASM) module using the tooling from [WebAssembly Hub](https://webassemblyhub.io) to simplify a lot of the packaging and sharing aspects. We'll then load the new module into a running Envoy Proxy and verify it works. Lastly, we'll look at the workflow to get the new module into the WebAssembly Hub via a pull-request workflow. 
+
+You can use your own distribution of Envoy that supports WebAssembly, but for this tutorial, we'll use Gloo which is an API Gateway based on Envoy. Gloo 1.0 announced beta support for WASM
+
+
+## Prepare environment
+
+To get started, let's deploy a sample service that we can call through Envoy. We'll deploy the sample petstore API:
+
+```shell
+$  kubectl apply -f \
+https://raw.githubusercontent.com/solo-io/gloo/master/example/petstore/petstore.yaml
+```
+
+You should now have the petstore running:
+
+```shell
+$  kubectl get po 
+
+NAME                        READY   STATUS    RESTARTS   AGE
+petstore-5dcf5d6b66-n8tjt   1/1     Running   0          2m20s
+```
+
+### Deploying Envoy
+
+In this tutorial, we'll use Gloo, an API Gateway based on Envoy that has built-in wasm support but these steps should also work for base Envoy. 
+
+```shell
+$  kubectl apply -f gloo.yaml
+```
+
+*****
+This section needs love... it's not working yet ^^ we need a nice way to install Gloo
```

Comment:
Can use helm for sure. The version of gloo will be `1.2.5`

Reply:
No, we don't want to use helm for this guide. We want to use as simple of a kube resource as possible. I'm trying to stay away from glooctl as well